src/BenchmarksServer/Startup.cs

```diff
@@ -174,36 +169,46 @@ public static int Main(string[] args)
             }
         }
 
-        private static void CloneAndRestore(string path, ServerJob job)
+        private static string CloneAndRestore(string path, ServerJob job)
         {
-            var benchmarksRepoUrl = string.IsNullOrEmpty(job.BenchmarksRepo) ? _defaultBenchmarksRepoUrl : job.BenchmarksRepo;
-            var benchmarksBranch = string.IsNullOrEmpty(job.BenchmarksBranch) ? _defaultBranch : job.BenchmarksBranch;
-            Git.Clone(path, benchmarksRepoUrl, benchmarksBranch, _benchmarksDir);
+            var benchmarksDir = Git.Clone(path, _benchmarksRepoUrl);
```

Comment:
It's nice to have meaningful directory names if you need to debug something, and the code to read from stderr is already written so it's easier to leave it at this point.